defaults/config.js

```diff
@@ -86,6 +86,28 @@ module.exports = {
 	//
 	prefetchStorage: false,
 
+	//
+	// Allow uploading files to the server hosting TheLounge.
+	// If enabled, The Lounge will store files in the `${THELOUNGE_HOME}/storage` folder.
+	//
+	// Images will not expire or be removed by TheLounge, so be wary of memory and your hardware.
+	//
+	// @type     boolean
+	// @default  false
+	//
+	upload: false,
+
+	//
+	// Maximum file size for uploaded files
+	//
+	// If upload is enabled, The Lounge will only allow uploading files with a size < the value below
+	// Specified value is in kilobytes. Default value is 10240 kilobytes (10 MB).
+	//
+	// @type     int
+	// @default  10240
+	//
+	uploadMaxFileSize: 10240,
```

Comment:
Can we allow this to be set to `-1` to be disabled?

Reply:
Previews dont have a max size, but sure.

Reply:
What do you mean?

Reply:
Previews have a max fetch size, but we have no way of setting unlimited. So this just isnt very consistent

Reply:
Ah right, well we can leave it for just now if we don't do it anywhere else.